spec.bs

```diff
@@ -2741,6 +2777,59 @@ CORP violation report=] algorithm, as leaving it unfenced may cause a privacy le
   [=automatic beacon data/once=] set to true.
 </div>
 
+<div algorithm=get-initiator-ancestor-root-origin>
+  To <dfn>get the initiator ancestor root origin</dfn> of a given {{Document}} |sourceDocument|:
```

Comment:
So this algorithm looks fine, but can we rename it something more specific? Maybe something super verbose like `get the ·initiator's fenced root's fenced root's origin`? I know that's wordy but it is crystal clear. WDYT? That or `get the initiator's fenced grand-ancestor's origin`.

Reply:
Changed to `get the initiator's fenced grand-ancestor's origin`.